### DISCUSSION TITLE
how to change frequency or use graphical interface remotly

### DISCUSSION THREAD

**Discussion by drepou** [rtl-airband/RTLSDR-Airband#325] [Q&A]
Is there any solution to changer the listen frequency in live or use a graphical UI to see and spy the bands like it's possible to do with sdrsharp over tcp server ??

I would like to use my key as a scanner and jump from frequency to frequency  until is i see something interrresting running so i dont know the "frequency" to listen before listen it...

**Comment by szpajder** [comment]
Nope, this program is designed for continuous unattended operation rather than for interactive wave surfing. Please just use your favorite graphical SDR console app for this purpose.
